tests/unit/models/util/ParseCalChart3File.spec.ts

```diff
@@ -0,0 +1,36 @@
+import Show from "@/models/Show";
```

Comment:
Do you think we need more unit tests for some of the helper functions? It might be good to make sure that it propagates errors correctly and that it parses the data correctly. You might have to create `DataView` objects and write some test data to them.

Reply:
We could, but this code isn't going to be used much I hope; I think we're going to be focused on the new stuff.

Reply:
Filed #49 to address this in the future.